### PR TITLE
Fix multi trigger issue

### PR DESCRIPTION
**Pull request by fancykingstar**
Fix the @ mentions to actually work. right now they don't work in the middle of a line. if you start a line with "@" it works but if you start typing text then type "@" it doesn't work.

### CODE REVIEW

**Comment by fancykingstar** [comment]
@fxOne Hi. Would you check the PR?

**Comment by fxOne** [comment]
Looks good to me!
Can you add your change to the changelog?

**Comment by fancykingstar** [comment]
Sure.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,3 +4,6 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Checkout the CHANGELOG.md of each package …
+
```
Comment (fxOne) [inline CHANGELOG.md:7]: Wrong file, please use the CHANGELOG.md in the package as described in the line above

**Comment by fancykingstar** [comment]
ok

**Comment by fancykingstar** [comment]
@fxOne I've added chage log again.
Thanks

**Review by fxOne — APPROVED** [review]

**Pull request merged**